google-translate-connector-test/src_test/com/axonivy/connector/google/translate/test/GoogleTranslateTest.java

```diff
@@ -0,0 +1,132 @@
+import ch.ivyteam.ivy.scripting.objects.CompositeObject;
+import google.translate.Language;
+
+@IvyProcessTest
+public class GoogleTranslateTest{
+
+  private static final BpmProcess TRANSLATE = BpmProcess.path("google/translate");
+
+  private interface Start {
+    BpmElement GOOGLE_TRANSLATE = TRANSLATE.elementName("getTranslatedText(String,String,String)");
+    BpmElement GOOGLE_LANGUAGES = TRANSLATE.elementName("getLanguages(String)");
+  }
+
+  @BeforeEach
+  void setup(AppFixture fixture, IApplication app) {
+    fixture.config("RestClients.googleTranslate.Url", GoogleTranslateServiceMock.URI);
+    RestClients clients = RestClients.of(app);
+    RestClient googleTranslate = clients.find("googleTranslate");
+    var testClient = googleTranslate.toBuilder()
+      .feature(CsrfHeaderFeature.class.getName())
+      .toRestClient();
+    clients.set(testClient);
+  }
+
+  @Test
+  public void translate1(BpmClient bpmClient){
+    ExecutionResult result = bpmClient.start()
+      .subProcess(Start.GOOGLE_TRANSLATE)
+      .withParam("text", "Hello World")
+      .withParam("targetLanguage", "de")
+      .withParam("sourceLanguate", "en")
+      .execute();
+    CompositeObject data = result.data().last();
+    try {
+		assertThat(data.get("translatedText")).isEqualTo("Hallo Welt");
+	} catch (NoSuchFieldException e) {
+		assertTrue(false);
+	}
+  }
+  
+  @Test
+  public void translate2(BpmClient bpmClient){
+    ExecutionResult result = bpmClient.start()
+      .subProcess(Start.GOOGLE_TRANSLATE)
+      .withParam("text", "Hello World")
+      .withParam("targetLanguage", "en")
+      .withParam("sourceLanguate", "en")
+      .execute();
+    CompositeObject data = result.data().last();
+    try {
+		assertThat(data.get("translatedText")).isEqualTo("Hello World");
+	} catch (NoSuchFieldException e) {
+		assertTrue(false);
+	}
+  }
+  
+  @Test
+  public void translate3(BpmClient bpmClient){
+    ExecutionResult result = bpmClient.start()
+      .subProcess(Start.GOOGLE_TRANSLATE)
+      .withParam("text", "Hello World")
+      .withParam("targetLanguage", "en")
+      .withParam("sourceLanguate", "")
+      .execute();
+    CompositeObject data = result.data().last();
+    try {
+		assertThat(data.get("translatedText")).isEqualTo("Hallo Welt");
+	} catch (NoSuchFieldException e) {
+		assertTrue(false);
+	}
+  }
+
+  @Test
+  public void languages1(BpmClient bpmClient){
+    ExecutionResult result = bpmClient.start()
+      .subProcess(Start.GOOGLE_LANGUAGES)
+      .withParam("targetLanguage", "en")
+      .execute();
+    CompositeObject data = result.data().last();
+    try {
+    	if (data.get("languages") != null) {
+    		var languages = (List<Language>) data.get("languages");
+    		assertTrue(languages.size() > 100);
+    	} else {
+    		assertTrue(false);
+    	}
+	} catch (NoSuchFieldException e) {
+		assertTrue(false);
+	}
+  }
+  
+  @Test
+  public void languages2(BpmClient bpmClient){
```

Comment:
same here: I think `languages_enToBlank` would share some information what the test is doing without actually making developers analysing the code.

Reply:
updated